HLTrigger/Muon/src/HLTMuonDimuonL3Filter.cc

```diff
@@ -161,19 +165,63 @@ HLTMuonDimuonL3Filter::hltFilter(edm::Event& iEvent, const edm::EventSetup& iSet
    Handle<RecoChargedCandidateCollection> mucands;
    if (saveTags()) filterproduct.addCollectionTag(candTag_);
    iEvent.getByToken(candToken_,mucands);
+
+   // Test to see if we can use L3MuonTrajectorySeeds:
+   if (mucands->size()<1) return false;
+   auto tk = (*mucands)[0].track();
```

Comment:
Yes its a TrackRef, ok I am testing with the const ref here and other suggestions below.